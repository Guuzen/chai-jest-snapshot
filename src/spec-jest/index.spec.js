import chai, { expect } from "chai";
import React from "react";
import renderer from "react/lib/ReactTestRenderer";

chai.use(require("../../dist/"))

const ExampleComponent = () => (
  <div>
    <h1>
      Hi!
    </h1>
  </div>
);
const tree = renderer.create(<ExampleComponent />).toJSON();
const OtherComponent = () => (
  <div>
    <h1>
      Hello!
    </h1>
  </div>
);
const otherTree = renderer.create(<OtherComponent />).toJSON();

describe("matchSnapshot", () => {

  describe("when running inside Jest", () => {

    it("should delegate to Jestâ€™s built-in matcher", () => {
      expect(tree).to.matchSnapshot()
    })

  })
});
