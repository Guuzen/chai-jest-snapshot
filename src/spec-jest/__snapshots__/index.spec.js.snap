// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matchSnapshot when running inside Jest should delegate to Jestâ€™s built-in matcher 1`] = `
<div>
  <h1>
    Hi!
  </h1>
</div>
`;
